### DISCUSSION TITLE
HA Core su Docker

### DISCUSSION THREAD

**Discussion by Dregi56** [indomus/forum#592] [HUB: Home Assistant]
Mi sono arenato sul più bello, caduto all'ultimo ostacolo.
![Cattura](https://user-images.githubusercontent.com/79653938/128577813-190d2f96-9fb4-43f9-89c9-4df7026bdfe1.JPG)

Il file docker-compose.yaml è lì, in quella directory, perchè non  lo trova?

**Comment by snakuzzo** [comment]
Se leggessi il messaggio d'errore che ti dà...lo capiresti. C'è scritto qual è il problema. Probabilmente non hai installato (o non hai installato correttamente) docker-compose

**Reply by Dregi56** [reply]
Grazie. Ciao
